crates/pathfinder/src/bin/pathfinder/config.rs

```diff
@@ -279,6 +293,8 @@ pub struct Config {
     pub python_subprocesses: std::num::NonZeroUsize,
     pub sqlite_wal: JournalMode,
     pub max_rpc_connections: std::num::NonZeroU32,
+    pub head_poll_interval: std::time::Duration,
+    pub pending_poll_interval: Option<std::time::Duration>,
```

Comment:
I think we can remove this item - we'll be leaving it disabled during the stress test. Although maybe no harm in having it..